authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHivePlugin.java

```diff
@@ -60,16 +81,286 @@ public Map<Privilege.Name, Set<RangerPrivilege>> privilegesMappingRule() {
   }
 
+  @Override
   /** Set Ranger policy resource rule. */
   public List<String> policyResourceDefinesRule() {
     return ImmutableList.of(
         PolicyResource.DATABASE.getName(),
         PolicyResource.TABLE.getName(),
         PolicyResource.COLUMN.getName());
   }
+
+  @Override
+  /** Allow privilege operation defines rule. */
+  public Set<Privilege.Name> allowPrivilegesRule() {
+    return ImmutableSet.of(
+        Privilege.Name.CREATE_CATALOG,
+        Privilege.Name.USE_CATALOG,
+        Privilege.Name.CREATE_SCHEMA,
+        Privilege.Name.USE_SCHEMA,
+        Privilege.Name.CREATE_TABLE,
+        Privilege.Name.MODIFY_TABLE,
+        Privilege.Name.SELECT_TABLE);
+  }
+
+  /** Translate the Gravitino securable object to the Ranger owner securable object. */
+  public List<RangerSecurableObject> translateOwner(MetadataObject metadataObject) {
+    List<RangerSecurableObject> rangerSecurableObjects = new ArrayList<>();
+
+    switch (metadataObject.type()) {
+      case METALAKE:
+      case CATALOG:
+        // Add `*` for the SCHEMA permission
+        rangerSecurableObjects.add(
+            RangerSecurableObjects.of(
+                ImmutableList.of(RangerHelper.RESOURCE_STAR),
+                MetadataObject.Type.SCHEMA,
+                ownerMappingRule()));
+        // Add `*.*` for the TABLE permission
+        rangerSecurableObjects.add(
+            RangerSecurableObjects.of(
+                ImmutableList.of(RangerHelper.RESOURCE_STAR, RangerHelper.RESOURCE_STAR),
+                MetadataObject.Type.TABLE,
+                ownerMappingRule()));
+        // Add `*.*.*` for the COLUMN permission
+        rangerSecurableObjects.add(
+            RangerSecurableObjects.of(
+                ImmutableList.of(
+                    RangerHelper.RESOURCE_STAR,
+                    RangerHelper.RESOURCE_STAR,
+                    RangerHelper.RESOURCE_STAR),
+                MetadataObject.Type.COLUMN,
+                ownerMappingRule()));
+        break;
+      case SCHEMA:
+        // Add `{schema}` for the SCHEMA permission
+        rangerSecurableObjects.add(
+            RangerSecurableObjects.of(
+                ImmutableList.of(metadataObject.name() /*Schema name*/),
+                MetadataObject.Type.SCHEMA,
+                ownerMappingRule()));
+        // Add `{schema}.*` for the TABLE permission
+        rangerSecurableObjects.add(
+            RangerSecurableObjects.of(
+                ImmutableList.of(metadataObject.name() /*Schema name*/, RangerHelper.RESOURCE_STAR),
+                MetadataObject.Type.TABLE,
+                ownerMappingRule()));
+        // Add `{schema}.*.*` for the COLUMN permission
+        rangerSecurableObjects.add(
+            RangerSecurableObjects.of(
+                ImmutableList.of(
+                    metadataObject.name() /*Schema name*/,
+                    RangerHelper.RESOURCE_STAR,
+                    RangerHelper.RESOURCE_STAR),
```

Comment:
How about  renaming`RESOURCE_START` to `RESOURCE_ALL`?

Reply:
DONE.